fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1703,6 +1703,30 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int iceberg_table_refresh_expire_sec = 86400;
 
+    /**
+     * iceberg fileio disk cache dir
+     */
+    @ConfField
+    public static String iceberg_fileio_cache_disk_medium_dir = StarRocksFE.STARROCKS_HOME_DIR + "/iceberg_metadata_cache";
```

Comment:
Because StarRocks has other caches like block cach in backend/compute node. How about we put all these types of caches in the same directory by default. 
like ${STARROCKS_HOME}/caches/iceberg/

Reply:
how about name it as `iceberg_metadata_cache_disk_path`

Reply:
done